src/appengine/private/components/jobs/multi-listbox.html

```diff
@@ -0,0 +1,81 @@
+<!--
```

Comment:
Shouldn't this go in components/common directory ? Also, might be nice to use it in other pages where we have multi list selection ?

Reply:
Actually multi-listbox is just a wrapper around paper-listbox with IronFormElementBehavior. This was only required in the jobs page as we were using iron-form. Other pages where multi-list selection is required uses ajax-requests which doesn't require the wrapper.